src/sentry/conf/server.py

```diff
@@ -2955,7 +2975,11 @@ def build_cdc_postgres_init_db_volume(settings):
     # Topic for indexer translated metrics
     KAFKA_SNUBA_METRICS: {"cluster": "default"},
     # Topic for receiving profiles from Relay
-    KAFKA_PROFILES: {"cluster": "default"},
+    KAFKA_PROFILES: {
+        "cluster": "default",
+        "group_id": "ingest-profiles",
+        "strategy_factory": "sentry.profiles.consumers.process.factory.ProcessProfileStrategyFactory",
```

Comment:
I think it'd be better if this was hardcoded rather than in configuration. I don't think the user should be able to change this value, and it seems like it would be easy to accidentally clobber it due to the way we override the whole "KAFKA_PROFILES" object in getsentry/ops.

Reply:
This structure also imposes a relationship between logical topics and consumers that we probably shouldn't. It should be possible to have 2 consumers connected to 1 logical topic, e.g. metrics indexer and last seen updater.

Reply:
yeah I realized the same. I changed it so that consumers are now registered by their consumer group instead.

Reply:
I'm not sure we should be registering by consumer group either. Consumer group is something that gets passed via the command line and can change frequently (e.g. during incidents). I think they should be hardcoded types with some other key that identifies them. In Snuba we use the concept of "storage" here where each writeable storage is associated with one consumer.

Reply:
Yeah i was thinking we could have a concept like a "logical consumer group". The physical can still be overridden in this pr